### DISCUSSION TITLE
Running your mod on the Minecraft launcher

### DISCUSSION THREAD

**Discussion by 12122234** [FabricMC/fabric-api#2515] [Mod Dev Support]
Today I created a mod, but I can only run it on vscode.
Any help?

**Comment by haykam821** [comment]
Is your mod being remapped properly? More information, such as logs, is needed to diagnose the issue.

**Reply by 12122234** [reply]
`Is your mod being remapped properly?`
What do you mean?

**Reply by haykam821** [reply]
Remapping is a process that makes mods compatible with the mappings used when running in production (intermediary) as opposed to development (Yarn, or another mapping set that you choose). Generally, Loom will do this automatically, but in some situations remapping may fail. The JAR suffixed with `-dev` is not remapped as well, so it will not work when running in production.
